### PR TITLE
Change layout of box 2 and box 51

### PR DESCRIPTION
**Pull request by BledarFeratiEY**

### CODE REVIEW

File: app/refactor/views/p4/tad/components/table_1/consignor_single.scala.xml [hunk 1]
```diff
@@ -8,33 +8,33 @@
...
     @vm.name
   </fo:block>
 </fo:table-cell>
 </fo:table-row>
 <fo:table-row height="5.33mm">
-<fo:table-cell display-align="center" padding-left="2mm"  font-weight="bold">
+<fo:table-cell display-align="center" padding-left="2mm" font-weight="bold">
   <fo:block>
-    @vm.city
+    @vm.streetAndNumber
```
Comment (joecefnhedog) [inline app/refactor/views/p4/tad/components/table_1/consignor_single.scala.xml:25]: are we concerned abut overflow?

Comment (joecefnhedog) [reply app/refactor/views/p4/tad/components/table_1/consignor_single.scala.xml:25]: image

Comment (joecefnhedog) [reply app/refactor/views/p4/tad/components/table_1/consignor_single.scala.xml:25]: this is the old design, but @vajarova had this problem:
image

Comment (BledarFeratiEY) [reply app/refactor/views/p4/tad/components/table_1/consignor_single.scala.xml:25]: image

Doesn't look too bad with the new format

**Review by joecefnhedog — APPROVED** [review]

**Pull request merged**